operations/validate.go

```diff
@@ -151,10 +158,9 @@ func validateFile(path string, ac *legacyClient, quiet, includeLong bool, localM
 
 // loadProjectIntoWithValidation returns a warning (instead of an error) if there's an error with unmarshalling strictly
 func loadProjectIntoWithValidation(ctx context.Context, data []byte, opts *model.GetProjectOpts,
-	project *model.Project) (*model.ParserProject, validator.ValidationErrors) {
+	project *model.Project) (*model.ParserProject, *model.ProjectConfig, validator.ValidationErrors) {
 	errs := validator.ValidationErrors{}
-	pp, _, err := model.LoadProjectInto(ctx, data, opts, "", project)
-
+	pp, pc, err := model.LoadProjectInto(ctx, data, opts, "", project)
```

Comment:
Do we only really need LoadProjectInto to return projectConfig for this function? In which case can we just separately unmarshal data into ProjectConfig as part of this function (using YamlWithFallback since any strict errors will still be caught on this line)? We'd still need to add ProjectConfig to the UnmarshalStrict struct but we wouldn't need to return the unmarshalled fields.

Reply:
Ok, I'll remove the project config unmarshalling from createIntermediateProject to decrease the amount of places we perform unmarshalling

Reply:
While I was at it I did the same for LoadProjectInto since a lot of places that call it don't care about ProjectConfig